index.bs

```diff
@@ -7814,16 +7816,24 @@ To <dfn export>run WebDriver BiDi preload scripts</dfn> given |environment setti
 
 1. Let |navigable| be |document|'s [=/navigable=].
 
+1. Let |user context| be |navigable|'s [=associated user context=].
+
+1. Let |user context id| be |user context|'s [=user context id=].
+
 1. For each |session| in [=active BiDi sessions=]:
 
   1. For each |preload script| in |session|'s [=preload script map=]'s
      [=values=]:
 
-    1. If |preload script|'s <code>contexts</code> is not null:
+    1. If |preload script|'s <code>user contexts</code>'s [=set/size=] is not zero:
+
+       1. If |preload script|'s <code>user contexts</code> does not [=list/contain=] |user context id|,
+          [=continue=].
 
-       1. Let |navigable id| be |navigable|’s [=navigable/top-level traversable=]'s id.
+    1. If |preload script|'s <code>contexts</code> is not null:
 
-       1. If |preload script|'s <code>contexts</code> does not [=list/contains|contain=] |navigable id|, continue.
+       1. If |preload script|'s <code>contexts</code> does not [=list/contain=] |user context id|,
+          [=continue=].
```

Comment:
This looks wrong. We still need to get the `navigable id` and check if it is in contexts.

Reply:
Indeed!